community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf

```diff
@@ -584,12 +584,17 @@ Use this database instead of the one on the controller.
   user      : The user to access the database as.
   password  : The password, given the user, to access the given database. (sensitive)
   db_name   : The database to access.
+  user_managed_replication : The list of location and (optional) kms_key_name for secret
 EOD
   type = object({
     server_ip = string
     user      = string
     password  = string # sensitive
     db_name   = string
+    user_managed_replication = list(object({
```

Comment:
@dgouju , I think this param should be `optional(list(...`, so it doesn't break previous usages that don't specify it.

Reply:
Ok, done!